build.rs

```diff
@@ -48,6 +50,12 @@ fn commit_hash() -> String {
     cmd.output().ok().and_then(|o| String::from_utf8(o.stdout).ok()).unwrap_or("None".to_owned())
 }
 
+fn branch_info() -> String {
+    let mut cmd = Command::new("git");
```

Comment:
Is this means the target machines should have git installed and the internet works.